python_modules/dagster/dagster/grpc/server.py

```diff
@@ -973,14 +985,15 @@ def open_server_process(
 
     mocked_system_timezone = get_mocked_system_timezone()
 
+    executable_path = loadable_target_origin.executable_path if loadable_target_origin else None
+
     subprocess_args = (
-        [
-            loadable_target_origin.executable_path
-            if loadable_target_origin and loadable_target_origin.executable_path
-            else sys.executable,
-            "-m",
-            "dagster.grpc",
-        ]
+        (
+            get_python_environment_entry_point(executable_path)
+            if executable_path
+            else DEFAULT_DAGSTER_ENTRY_POINT
+        )
+        + ["api", "grpc"]
```

Comment:
i got confused here for a bit wondering where entry point was 

the distinction between LoadableTargetOrigin only having executable_path and the other having both is subtle